docs/reference/logging/overview.md

```diff
@@ -90,16 +107,36 @@ the log tag format.
 
 ## journald options
 
+The journald logging driver supports the following options:
+
+    --log-opt labels=label1,label2
+    --log-opt env=env1,env2
+
 The `journald` logging driver stores the container id in the journal's `CONTAINER_ID` field. For detailed information on
 working with this logging driver, see [the journald logging driver](journald.md)
 reference documentation.
 
+The `labels` and `env` options adds additional attributes to be logged
```

Comment:
Well, for each driver the representation differs :

- `json-file` : goes in an `attrs` attribute in the json log file
- `journald` : add log fields to the current logs (something like tags.. I think :sweat_smile:)
- `fluentd` : add entries for each keys in the `extra` attribute
- `gelf` : almost the same as `fluentd`

That's why I repeated it (and the first description is in `json-file`).

I think the first description should go above then (in a `generic` options) and have a small paragraph on the *representation* for each driver that supports it ? @moxiegirl WDYT ?